### DISCUSSION TITLE
Rotation property

### DISCUSSION THREAD

**Discussion by guerinoni** [slint-ui/slint#709] [General] [answered]
Hello, I'm searching for a property like `rotation` for Qt users :)
Am I missing some resources, or is this not still supported?

**Comment by ogoffart** [comment, marked as answer]
We started to implement that but did not finish.
There is an internal "Rotate" item, bit it is not exposed in the public API. 

The goal is to detect "rotation" property and transform it to that element, just like what we do for the opacity or the clip.

If you remove that line: https://github.com/sixtyfpsui/sixtyfps/blob/ffe561f8d801b39ac804d229508844399777aa3c/sixtyfps_compiler/builtins.60#L78 
You'd be able to use the Rotate element in your code. (But there might be bugs)

**Reply by guerinoni** [reply]
Many thank, I'm working on an early software for embedded platform, I can handle some bugs :)
